apps/condo/domains/acquiring/schema/RegisterMultiPaymentService.test.js

```diff
@@ -1281,7 +1315,8 @@ describe('RegisterMultiPaymentService', () => {
             })
         })
 
-        test('Should correctly register multiPayment for receipt and invoice', async () => {
+        // Skip this test until we allow to create the multi-payment which including both, invoices and receipts
```

Comment:
Maybe convert this to TODO and leave task number?

Reply:
Done